app/controllers/smart_proxies_controller.rb

```diff
@@ -24,7 +27,6 @@ def edit
   end
 
   def ping
-    @proxy = @smart_proxy
```

Comment:
This was added in https://github.com/theforeman/foreman/commit/8b737c9c7648b3726dadb3b2e4708fcb43af02a8 
I think that `Parameterizable::ByIdName` overrides it anyhow